pages/common/nslookup.md

```diff
@@ -0,0 +1,27 @@
+# nslookup
+
+> Query name server(s) for various domain records.
+
+- Query your system's default name server for an IP address (A record) of the domain:
+
+`nslookup {{example.com}}`
+
+- Query non default name server (here Google DNS 8.8.8.8) for a NS record of the domain:
+
+`nslookup -type=NS {{example.com}} 8.8.8.8`
+
+- Query default name server for a reverse lookup (PTR record) of an IP address:
+
+`nslookup -type=PTR 54.240.162.118`
+
+- Query a given name server for the whole zone file (zone transfer) of the domain using TCP protocol (-vc option) instead of the default UDP:
+
+`nslookup -vc -type=AXFR {{example.com}} {{name_server}}`
+
+- Query default name server for a mail server (MX record) of the domain, showing details of the transaction:
```

Comment:
You don't need to say "default name server" every time. After 1 or 2 examples, its clear enough.